mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_2794 net/minecraft/world/gen/chunk/ChunkGenerator
+	COMMENT In charge of shaping, adding biome specific surface blocks, carving, and decorating chunks,
+	COMMENT as well as populating {@linkplain net.minecraft.world.biome.Biome biomes} and {@linkplain net.minecraft.entity.Entity entities}.
```

Comment:
In summary, basically everything related to generation.
I would change "populating biomes and entities" to "populating features and entities" because I believe placing entities is part of biome population, and features is more specific.

Reply:
Yeah. Often the actual biome placement is done with a `BiomeSource` but the biome source stuff is called by the chunk generator's `populateBiomes` method. Might add a note about that.